.github/workflows/unit_test_reusable.yml

```
@@ -37,3 +37,26 @@ jobs:
       - name: Run frontend unit tests
         working-directory: ./frontend
         run: yarn test
+
+  e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+          cache-dependency-path: '**/yarn.lock'
+      - name: Install frontend dependencies
+        working-directory: frontend
+        run: yarn install --frozen-lockfile
+      # Stand up the system stack, to have something to poke
+      - name: Start Stack
+        run: docker-compose up --build -d
+      # Run the Cypress E2E Tests
+      - name: E2E Test
+        working-directory: frontend
+        run: yarn run test:e2e
```

A little thing that `yarn` does that `npm` doesn't is not require `run`.

```suggestion
        run: yarn test:e2e
```